tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -0,0 +1,109 @@
+"""Unit tests for Riemannian Mean Shift method."""
+
+import os
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.hypersphere import Hypersphere, HypersphereMetric
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+from geomstats.learning.frechet_mean import FrechetMean
+from geomstats.learning.riemannian_mean_shift import (
+    RiemannianMeanShift as riemannian_mean_shift,
+)
+
+os.environ["GEOMSTATS_BACKEND"] = "pytorch"
```

Comment:
You should remove this line, because this file should be able to run for all of the backends: we don't want a specific backend to get hardcoded here.